chart/values.yaml

```diff
@@ -220,6 +220,21 @@ secret: []
 #   secretName: ""
 #   secretKey: ""
 
+# Enables selected built-in secrets that are set via environment variables by default.
+# Those secrets are provided by the Helm Chart secrets by default but in some cases you
+# might want to provide some of those variables with _CMD or _SECRET variable, and you should
+# in this case disable setting of those variables by setting the relevant configuration to false.
```

Comment:
We should probably add something in https://airflow.apache.org/docs/helm-chart/stable/production-guide.html too -- on "How to use Secrets backend" that involves setting one of the following as `False` and passing another env var for the actual string?

Reply:
Good Point.